lib/menu.rb

```diff
@@ -0,0 +1,50 @@
+require 'csv'
+
+class Menu
+  attr_reader :dishes, :available_dishes
+
+  def initialize(dishes_file = "./assets/dishes.csv")
+    @dishes = []
+    load_dishes(dishes_file)
+
+    @available_dishes = filter_dishes_by_available
+  end
+
+  def display_available_dishes
+    available_dishes
+  end
+
+  private
+
+  attr_writer :dishes, :available_dishes
+
+  def load_dishes(dishes_file)
+    CSV.foreach(dishes_file, headers: true, header_converters: :symbol) do |row|
+      name, price, available = row[:name], row[:price], row[:available]
+
+      @dishes << { name: name.to_sym, price: price.to_f, available: true?(available) }
+    end
+  end
+
+  def true?(available)
+    available == "true"
+  end
+
+  def filter_dishes_by_available
+    self.available_dishes = dishes.select do |dish| 
+      dish[:available] == true
```

Comment:
The functionality around whether a dish is available or not is adding a slight degree of complexity that isn't asked for in the spec - do try to keep your implementation as simple as possible by following the spec closely. I think the 'available'ness mentioned in the user story simply refers to dishes that are on sale at the restaurant.